tests/test_async_query.py

```diff
@@ -243,15 +243,21 @@ async def test_async_query_single_command_03(self):
 
         with self.assertRaisesRegex(
                 edgedb.InterfaceError,
-                r'cannot be executed with query_single\(\).*'
+                r'cannot be executed with query_required_single\(\).*'
                 r'not return'):
-            await self.con.query_single('START TRANSACTION')
+            await self.con.query_required_single('START TRANSACTION')
 
-        with self.assertRaisesRegex(
-                edgedb.InterfaceError,
-                r'cannot be executed with query_single_json\(\).*'
-                r'not return'):
-            await self.con.query_single_json('START TRANSACTION')
+        # self.assertEqual(
+        #     await self.con.query_single('START TRANSACTION'), None)
+
+        # with self.assertRaisesRegex(
```

Comment:
uncomment?

Reply:
I left this bit commented out for now, because of this bug: https://github.com/edgedb/edgedb/issues/3120

Reply:
Let's reference this but in the comment then.